### PR TITLE
Update setup-client-windows.html

### PR DESCRIPTION
**Pull request by EpicTaru**
Final pass of grammar/formatting/clarity changes, no other changes will be necessary with relation to all these PRs of this page as of late.

### CODE REVIEW

File: home/setup-client-windows.html [hunk 1]
```diff
@@ -12,28 +12,31 @@ <h2>Windows Installation</h2>
...
-  <li>At least 10GB available.</li>
-  <li>Know how to use Windows File Explorer.</li>
-  <li>Know how to run .exe files as an administrator.</li>
-  <li>Know how to use Windows Powershell as an administrator.</li>
+  <li>Fast internet connection</li>
+  <li>Admin rights on the machine</li>
+  <li>At least 10GB available</li>
+  <li>Know how to use Windows File Explorer</li>
+  <li>Know how to run .exe files as an administrator</li>
+  <li>Know how to use Windows Powershell as an administrator</li>
```
Comment (kaincenteno) [inline home/setup-client-windows.html:20]: why removed the "." at the end?

Besides this everything looks awesome. thanks for going through it taru <3

Comment (EpicTaru) [reply home/setup-client-windows.html:20]: I is grammar police, for a list of requirements, period not required unless item in list is a complete sentence

Comment (EpicTaru) [reply home/setup-client-windows.html:20]: > I is grammar police, for a list of requirements, period not required unless item in list is a complete sentence

And even this is debatable, depends on context too, lol

Comment (kaincenteno) [reply home/setup-client-windows.html:20]: lol thank you grammar police xD

**Pull request merged**